src-docs/src/views/empty_prompt/empty_prompt_layout.tsx

```diff
@@ -14,7 +14,7 @@ export default () => {
   return (
     <>
       <EuiEmptyPrompt
-        icon={<EuiImage size="fullWidth" src={illustration} alt="" />}
+        icon={<EuiImage size="l" src={illustration} alt="" />}
```

Comment:
```suggestion
        icon={<EuiImage size="fullWidth" src={illustration} alt="" />}
```


The other illustrations have the size `"fullWidth"`. Also we're saying in our docs:

<img width="784" alt="Screenshot 2022-03-01 at 11 17 53" src="https://user-images.githubusercontent.com/2750668/156159830-f1a3c769-778f-494c-9874-575c3dac60f3.png">